client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -46,61 +48,74 @@ const DesignSelector: React.FunctionComponent = () => {
 				<div className="design-selector__heading">
 					<Title>{ __( 'Choose a design' ) }</Title>
 					<SubTitle>
-						{ __( 'Pick your favorite homepage layout. You can customize or change it later.' ) }
+						{ isAnchorFmSignup
+							? __(
+									'Pick a homepage layout for your podcast site. You can customize or change it later.'
+							  )
+							: __( 'Pick your favorite homepage layout. You can customize or change it later.' ) }
 					</SubTitle>
 				</div>
 				<ActionButtons>
 					<BackButton onClick={ goBack } />
 				</ActionButtons>
 			</div>
 			<div className="design-selector__design-grid">
-				<div className="design-selector__grid">
-					{ getRandomizedDesigns().featured.map( ( design ) => (
-						<button
-							key={ design.slug }
-							className="design-selector__design-option"
-							data-e2e-button={ design.is_premium ? 'paidOption' : 'freeOption' }
-							onClick={ () => {
-								setSelectedDesign( design );
+				<div
+					className={ isAnchorFmSignup ? 'design-selector__grid-minimal' : 'design-selector__grid' }
+				>
+					{ getRandomizedDesigns()
+						.featured.filter(
+							( design ) =>
+								// TODO Add finalized design templates to client/landing/gutenboarding/available-designs-config.json
+								// along with is_anchorfm prop
+								isAnchorFmSignup === ( design.is_anchorfm ?? false )
```

Comment:
What's the difference between this and `isAnchorFmSignup && design.is_anchorfm`?

Reply:
Consider the case when `isAnchorFmSignup` is false, and `design.is_anchorfm` is false or undefined.

* `isAnchorFmSignup && design.is_anchorfm` evaluates to false.
  * Thus, in the non-anchor flow, no designs are shown.
* `isAnchorFmSignup === ( design.is_anchorfm ?? false )` evaluates to true.
  * Thus, in the non-anchor flow, all of the non-anchor designs are shown.

Reply:
You're right -- I thought about this just as I fell asleep 😴